### DISCUSSION TITLE
Recommended antenna

### DISCUSSION THREAD

**Discussion by ghost** [szpajder/dumphfdl#28] [Q&A]
Hello,
What antenna would you recommend for use with dumphfdl? What antenna do you use?

Thanks

**Comment by szpajder** [comment]
See here:

https://github.com/szpajder/dumphfdl#what-antenna-should-i-use

I use a Wellbrook loop because I'm in a city and I don't have space for a full-size wire antenna (which would work as well, possibly even better, if installed in a quiet location).

**Comment by jaymot123** [comment]
I mostly use a K-180WLA active loop antenna from China, with its included RG174 coax replaced by some LMR195 I happened to have (RG58 would be fine too) and the switch that's in its amplifier changed from the default FM-DX setting to HF.  Sometimes I switch to a cobbled-together dipole on top of my fence whose wires have been looped back and forth, along with a 1:1 balun.
